packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -100,6 +102,7 @@ oracleAdaptersProxy = "<%= imports.reyaOracleAdaptersProxy.contracts.InitialProx
 oracleManagerProxy = "0xC67316Ed17E0C793041CFE12F674af250a294aab"
 exchangePassProxy = "0x76e3f2667aC55d502e26e59C5A6B46e7079217c7"
 ranksProxy = "0x4Ab90747A2E8dEC7C378D591b80B9f2f8f78351F"
+sbtProxy = ""
```

Comment:
should be the same as cronos

Reply:
fixed in [a2be103](https://github.com/Reya-Labs/reya-deployments/pull/259/commits/a2be103ccbf73d6d1e2bb8359e9549f627b41f10)